Issues with npm build on new clone

There's a good chance this is just me having this issue, but wanted to document it.

Brand new clone:

```bash
▲ nvm use
Found '/Users/brad/Sites/sandbox/content/themes/wd_s/.nvmrc' with version <14>
Now using node v14.17.1 (npm v6.14.13)

▲ npm i --legacy-peer-deps
npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @wordpress/prettier-config@2.2.1-beta-1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'wd_s'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/brad/.npm/_logs/2021-06-16T00_23_17_784Z-debug.log
```

oh wait, I don't know if `nvm` used the right version for this.

```bash
▲ nvm install && nvm use
...

▲ npm i --legacy-peer-deps

npm ERR! code ETARGET
npm ERR! notarget No matching version found for @wordpress/prettier-config@2.2.1-beta-1.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of 'wd_s'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/brad/.npm/_logs/2021-06-16T00_40_08_646Z-debug.log
```

Changing `"prettier": "npm:@wordpress/prettier-config@2.2.1-beta-1",` to `"prettier": "npm:@wordpress/prettier-config",` in package.json gives me:

```bash
> wd_s@2.1.0 build /Users/brad/Sites/sandbox/content/themes/wd_s
> cross-env NODE_ENV=production wp-scripts build --config webpack.prod.js

Hash: f98570591dd383c3484c
Version: webpack 4.46.0
Time: 44570ms
Built at: 06/15/2021 7:41:50 PM
 14 assets
Entrypoint index = index.css index.css index.js
[10] ./src/index.js + 2 modules 931 bytes {0} [built]
     | ./src/index.js 781 bytes [built]
     | ./src/scss/tailwind.scss 50 bytes [built]
     | ./src/scss/index.scss 50 bytes [built]
    + 12 hidden modules

ERROR in Cannot find module 'prettier'
Require stack:
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint-plugin-prettier/eslint-plugin-prettier.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/@eslint/eslintrc/lib/config-array-factory.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/@eslint/eslintrc/lib/index.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint/lib/cli-engine/cli-engine.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint/lib/cli-engine/index.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint/lib/api.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint-webpack-plugin/dist/getESLint.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint-webpack-plugin/dist/linter.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint-webpack-plugin/dist/index.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/eslint-webpack-plugin/dist/cjs.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/webpack.config.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/webpack.prod.js
- /Users/brad/Sites/sandbox/content/themes/wd_s/node_modules/webpack-cli/bin/utils/convert-argv.js
```

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
 		"npm-run-all": "^4.1.5",
 		"postcss-loader": "4.2.0",
 		"postcss-preset-env": "^6.7.0",
-		"prettier": "npm:@wordpress/prettier-config@2.2.1-beta-1",
+		"prettier": "npm:@wordpress/prettier-config",
 		"sass-loader": "10.2.0",
 		"stylelint-config-prettier": "^8.0.2",
 		"stylelint-webpack-plugin": "^2.1.1",

